package.json

```diff
@@ -9,6 +9,7 @@
         "lint": "tslint ./src/**/*.{ts,tsx}",
         "clean": "rm -rf ./dist ./node_modules",
         "build": "tsc",
+        "format": "prettier --write \"**/*.{ts,tsx,json,md}\"",
```

Comment:
What do you think about running this on prepublish?

Reply:
We should better have that as commit hook for the future, but doing that before publishing actually could create another version of the code as it is in the repo. So I would not go with that approach.